api/client_test.go

```diff
@@ -73,6 +73,13 @@ func TestHttpClient_OrganizationById(t *testing.T) {
 			t.Errorf("Expected return organization identifier to be [%s], got [%s]", organizations[0].Identifier, res.Identifier)
 		}
 	})
+
+	t.Run("http execution error", func(t *testing.T) {
```

Comment:
Is it needed to do this test 6 times?

Reply:
Code (call HTTP client, check err) is there 6 times (actually more) and we needed more project test coverage, so yes... :-(